app/uk/gov/hmrc/helptosavefrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -43,7 +43,9 @@ trait EmailVerificationConnector {
 class EmailVerificationConnectorImpl @Inject() (http: WSHttp, conf: Configuration)(implicit crypto: Crypto)
```

Comment:
we dont need to inject `conf` when you had already extended `ServicesConfig`

Reply:
true